cmd/entrypoints/util/token_cache_keyring.go

```diff
@@ -0,0 +1,58 @@
+package pkce
+
```

Comment:
May be we can move this from flytectl to stdlib so that admin can consume aswell?

Reply:
thanks for the callout, removed!

Reply:
Still see this...

Reply:
sorry, removed the references to it but forgot to delete. mind taking another look?